extensions/security/deployment/src/main/java/io/quarkus/security/deployment/SecurityProcessor.java

```diff
@@ -214,6 +214,9 @@ private static void prepareBouncyCastleProvider(CurateOutcomeBuildItem curateOut
                     .produce(new RuntimeReinitializedClassBuildItem("org.bouncycastle.jcajce.provider.drbg.DRBG$Default"));
             runtimeReInitialized
                     .produce(new RuntimeReinitializedClassBuildItem("org.bouncycastle.jcajce.provider.drbg.DRBG$NonceAndIV"));
+            runtimeReInitialized
```

Comment:
Could you please add a comment to the fact as to why we need to runtime-reinit this class? Something like:

```
// URLSeededEntropySourceProvider.seedStream may contain a reference to a 'FileInputStream' which includes
// references to FileDescriptors which aren't allowed in the image heap
```

Thanks!

Reply:
Good idea, thanks